### DISCUSSION TITLE
Has anyone here successfully used GET_LEDGER_SUMMARY_VIEW_DATA and can help me?

### DISCUSSION THREAD

**Discussion by nordinho** [amzn/selling-partner-api-models#3447] [General] [answered]
I've been trying to get the inventory ledger report with this code but it only returns an empty document with only the column names:

report_type = ReportType.GET_LEDGER_SUMMARY_VIEW_DATA
res_reports = Reports(credentials=credentials, marketplace=Marketplaces.US).create_report(reportType=report_type,reportOptions={"aggregateByLocation":"COUNTRY"})
 I've tried it with reportOptions and without.
 
 The document I download only has this:

"Date"	"FNSKU"	"ASIN"	"MSKU"	"Title"	"Disposition"	"Starting Warehouse Balance"	"In Transit Between Warehouses"	"Receipts"	"Customer Shipments"	"Customer Returns"	"Vendor Returns"	"Warehouse Transfer In/Out"	"Found"	"Lost"	"Damaged"	"Disposed"	"Other Events"	"Ending Warehouse Balance"	"Unknown Events"	"Location"

**Comment by nordinho** [comment, marked as answer]
I figured out the issue, for anyone else who's having problems with this you need to add dataStartTime and dataEndTime  in ISO 8601 date time format.

**Reply by NA-Dev** [reply]
Thank you! That fixed it!

**Reply by yashraj021** [reply]
Can you paste the example

**Comment by mchaelkha** [comment]
Another resolution to this may be how you are specifying the dates to conform to aggregatedByTimePeriod reportOptions. In my case, I was trying to generate daily aggregate reports, but by default the report is set to monthly causing no data to be returned.

**Reply by yashraj021** [reply]
Hi, can you paste a sample of payload?
